src/Http/TreeRouteStack.php

```diff
@@ -242,8 +241,8 @@ protected function routeFromArray($specs)
     /**
      * Add multiple prototypes at once.
      *
-     * @param  Traversable $routes
-     * @return TreeRouteStack
+     * @param array<string|array|Traversable|TRoute>|Traversable<string|array|Traversable|TRoute> $routes
```

Comment:
```suggestion
     * @param iterable<string|iterable|TRoute> $routes
```
as `iterable` is an alias for `array|Traversable` https://www.php.net/manual/en/language.types.iterable.php

Reply:
58aa9bfcde673252fe98629c3067838ffe1ba62b